README.rst

```diff
@@ -178,6 +178,11 @@ In this case, extra *docker run* options can be provided in your *"registry:look
 
 By default, the storage backend used by the registry is "filesystem". Use environment variables to override that, for example to use S3 as backend storage.
 
+Development
+===========
+
+Note that some of the internal states such as `docker.running` are references to the internal `dockerio states <https://docs.saltstack.com/en/latest/ref/states/all/salt.states.dockerio.html>`
```

Comment:
All of the modules referenced in here:
https://github.com/saltstack-formulas/docker-formula/blob/master/docker/compose-ng.sls are dockerio state modules.
```
docker.pulled
docker.installed
docker.running
```
Where else would they be coming from? Search for `pulled` in the repo, and you get one hit with this file. What would `pulled` refer to if it was not `dockerio` state module?